app/assets/images/wifi_icon.svg

```diff
@@ -0,0 +1,85 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
+<!-- Created with Inkscape (http://www.inkscape.org/) -->
+
+<svg
+   xmlns:dc="http://purl.org/dc/elements/1.1/"
```

Comment:
svgs should be minimized before we add them to the project. Check out a tool like this one to turn it into a regular jsx component; I think it minimizes also

https://svg2jsx.com/